docs/content/latest/explore/ysql-language-features/going-beyond-sql/follower-reads-ycql.md

```diff
@@ -8,10 +8,11 @@ aliases:
   - /latest/explore/tunable-reads/
   - /latest/explore/follower-reads/
   - /latest/explore/high-performance/tunable-reads/
+  - /latest/explore/multi-region-deployments/follower-reads-ycql/
 menu:
   latest:
-    identifier: explore-multi-region-deployments-follower-reads-ycql
-    parent: explore-multi-region-deployments
+    identifier: going-beyond-sql-follower-reads-ycql
```

Comment:
Might be good to shorten the identifier to `follower-reads-ycql` (and do the same for ysql). If nothing else, it'll be easier to move it some other time :)

Reply:
Understood. Will shorten it. Thanks.